### ISSUE TITLE
Split checkpoint should allow players to switch sides

### ISSUE THREAD

**Issue by ncrecc** [BEEmod/BEE2-items#2134]
Normally the split-off/separated checkpoint locks the players on the sides they choose. However, the co-op campaign's Art Therapy 01 allows the players to switch sides, while still requiring them to both be on different sides in order to get past the airlock. The developer commentary even brings up how "by swapping roles, each player can get an entirely different experience within the exact same puzzle."

**Comment by TeamSpen210** [comment]
I'm not sure what you mean? It already allows this. Unless you mean 'undoing' your choice, and swapping again after?

**Comment by ncrecc** [comment]
Yes, that's what I mean. You should be able to switch sides after the spawn points are changed.

**Comment by Endermage77** [comment]
In that case, you could simply add an option to force the checkpoint's entry/exit doors open after you enter: That would be the simplest way to go about it in my mind.

**Comment by TeamSpen210** [comment]
I checked the map, and actually it does not do this. (I should check the second part though, maybe that does?)
